packages/nc-gui/components/createOrEditProject.vue

```diff
@@ -1403,6 +1394,7 @@ export default {
         this.projectCreated = true
       } catch (e) {
         this.$toast.error(await this._extractSdkResponseErrorMsg(e)).goAway(3000)
+        toast.goAway(100)
```

Comment:
Suggest to make it `0` instead of `100` so that we don't see the blue toast just like in xcdb one

Reply:
updated